File: app/src/test/java/com/mss/imagesearcher/di/RobolectricApp.java
package com.mss.imagesearcher.di;

import com.mss.imagesearcher.App;

public class RobolectricApp extends App {

    @Override
    protected void initAppComponent() {
        Companion.setAppComponent(DaggerRobolectricComponent
                .builder()
                .build());
    }

    @Override
    protected void initDebug() {
    }
}

File: app/src/test/java/com/mss/imagesearcher/di/RobolectricApp.kt
package com.mss.imagesearcher.di

import com.mss.imagesearcher.App

class RobolectricApp : App() {

    override fun initAppComponent() {
        appComponent = DaggerRobolectricComponent
                .builder()
                .build()
    }

    override fun initDebug() {}
}
